src/Header.js

```diff
@@ -24,12 +25,23 @@ const Header = ({ userID, handleSignout }) => {
     }
   };
 
+  if(userID === "") {
+    bannerComponent = 
+      <Link to="/">
+        <img src={banner} id="banner" alt="Mustang Messenger banner" />
+      </Link>
+    ;
+  }
+  else {
+    bannerComponent = 
+      <img src={banner} id="banner" alt="Mustang Messenger banner" />
+    ;
+  }
+
```

Comment:
What's the if doing here other than making the banner a link when the user is signed in? Should we just have consistent behavior for both cases?

Reply:
This is because I wanted to make the landing page inaccessible (by clicking, at least) to the user when they're already signed in. Logically they wouldn't need to read the landing page if they are already a registered user. If you think it should be consistent behavior for both cases, let me know and I can make the appropriate changes.

Reply:
That makes sense, lets keep it like that.